src/bp-activity/bp-activity-filters.php

```diff
@@ -2997,3 +3000,28 @@ function bb_activity_delete_link_review_attachment( $activities ) {
 		}
 	}
 }
+
+/**
+ * Wrap with "P" tag if found the single plan text link.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param string|null $content Activity content.
+ *
+ * @return string
+ */
+function bb_activity_content_has_paragraph_tag( $content = '' ) {
+
+	if ( empty( $content ) ) {
+		return $content;
+	}
+
+	preg_match( '/(https?:\/\/[^\s<>"]+)/i', $content, $content_url );
+	preg_match( '(<p(>|\s+[^>]*>).*?<\/p>)', $content, $content_tag );
+
+	if ( ! empty( $content_url ) && empty( $content_tag ) ) {
+		$content = sprintf( '<p>%s</p>', $content );
+	}
+
+	return $content;
+}
```

Comment:
Remove this code from here.
add it on: https://github.com/buddyboss/buddyboss-platform/blob/release/src/bp-core/classes/class-bp-embed.php#L299

```
if ( ! empty( $embed_code ) ) {

	if ( ! empty( $content ) ) {
		preg_match( '/(https?:\/\/[^\s<>"]+)/i', $content, $content_url );
		preg_match( '(<p(>|\s+[^>]*>).*?<\/p>)', $content, $content_tag );

		if ( ! empty( $content_url ) && empty( $content_tag ) ) {
			$content = sprintf( '<p>%s</p>', $content );
		}
	}

	$content .= $embed_code;
	$link_embed = true;
}
```